airbyte-integrations/connectors/source-appfollow/source_appfollow/manifest.yaml

```diff
@@ -0,0 +1,45 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.appfollow.io"
+    http_method: "GET"
+    authenticator:
+      type: "BasicHttpAuthenticator"
+      username: "{{ config['api_secret'] }}"
+      password: ""
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```

Comment:
Pagination needs to be configured. For example:

```
base_paginator:
    type: "DefaultPaginator"
    pagination_strategy:
      type: "CursorPagination"
      cursor_value: "{{ last_records['nextPageToken'] }}"
    page_token_option:
      type: "RequestPath"
      field_name: "pageToken"
      inject_into: "request_parameter"
```

See [the docs](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/pagination) for more information.